app/components/title-input.js

```diff
@@ -9,13 +9,30 @@ export default Ember.Component.extend({
     this.$('input').focus();
   },
 
+  enter() {
+    this.$('input').blur();
+  },
+
   actions: {
     inputFocusIn() {
+      if(this.$('input').val()){
+        this.$('input').val('');
+    }
```

Comment:
1. Please fix indentation.
2. Won't this delete a title that isn't the default text?